app/models/user.rb

```diff
@@ -86,7 +86,7 @@ def assign_barcode
   # error message if PIN is invalid:
   # "PIN is not valid : PIN is trivial"
   def validate_pin_pattern
-    if pin.scan(/(.)\1{2,}/).empty? && pin.scan(/(..)\1{1,}/).empty? == true
+    if pin && pin.scan(/(.)\1{2,}/).empty? && pin.scan(/(..)\1{1,}/).empty? == true
```

Comment:
Let's add the "non-trivial pin" validation, so admins don't input "1111" (see regex on Raj's user.rb").

Reply:
@daryanypl it looks like the validation is already happening.
<img width="957" alt="screen shot 2018-09-11 at 12 38 02 pm" src="https://user-images.githubusercontent.com/1825103/45374286-99e36e00-b5bf-11e8-84e8-712d967028cd.png">

Reply:
Thanks @daryanypl here's what it looks like now:
<img width="970" alt="screen shot 2018-09-12 at 2 30 49 pm" src="https://user-images.githubusercontent.com/1825103/45445710-f8304f80-b698-11e8-915d-ddfe1e873d91.png">